setup.py

```diff
@@ -1,14 +1,12 @@
 import setuptools
 
-import embedchain.version
```

Comment:
from embedchain import version

Reply:
hi @chilkaditya thats a good catch. Do you want to create a PR to fix this?

Reply:
This is fixed in #280. 

```
import embedchain as ec
print(ec.__version__)

```